wiki/Miscellaneous_official_projects/upppy/en.md

```diff
@@ -0,0 +1,42 @@
+---
+tags:
+  - upppy
+  - up.ppy
+  - up.ppy.sh
+---
+
+# upppy
+
+::: Infobox
+![Snapshot of the up.ppy.sh page](img/upppyScreenshot.png "Snapshot of the upppy website, as taken in 2023 by the Internet Archive")
+:::
+
+**up.ppy.sh**, often referred to as **upppy**, was a file uploading service ran and hosted by ::{ flag=AU }:: [peppy](https://osu.ppy.sh/users/2). It was first established in January 2007, predating osu! itself[^upppy-launch-date-ref], and was mainly used to share images and files within the osu! community.
+
+## Service
+
+upppy was created as a free, lightweight alternative to proprietary image hosting services such as Rapidshare or ImageShack.[^upppy-image-hosting-ref]
+
+The service didn't require any registration to use and allowed for everyone to upload any file (within its Terms of Service) anonymously. Similar to certain [imageboard sites](https://en.wikipedia.org/wiki/Imageboard), upon uploading a file, users were given a choice as to whether to post their file in the default, catch-all working channel (`#random`) or in the one of the more specific channels as follows:
+
+   - `#anime`
+   - `#game`
+   - `#highres`
+   - `#music`
+   - `#photography`
+   - `#screenshots`
+   - `#torrents`
+   - `#tv`
+   - `#video`
+   - `#wallpaper`
+
+Once a file was posted, it became a permanent part of the platform and couldn't be deleted or edited by the uploader. However, much like on [Reddit](/wiki/Community/Reddit), these posts could still receive comments and be upvoted/downvoted by other users.
```

Comment:
the second sentence reads as if it is rather unexpected -- perhaps remove `However`?

Reply:
hmm i personally find the sentence to be easier to read with the "however" there 😦